src/cmap/connection_pool.ts

```diff
@@ -234,6 +236,7 @@ export class ConnectionPool extends TypedEventEmitter<ConnectionPoolEvents> {
       maxConnecting: options.maxConnecting ?? 2,
       maxIdleTimeMS: options.maxIdleTimeMS ?? 0,
       waitQueueTimeoutMS: options.waitQueueTimeoutMS ?? 0,
+      minPoolSizeCheckIntervalMS: options.minPoolSizeCheckIntervalMS ?? 100,
```

Comment:
100 is arbitrary; this is left entirely to the drivers' discretion in the spec, but we were using `10` and `100` before, depending on the circumstance, and `100` seemed frequent enough

Reply:
Yea 100 sounds good